samples/nodefeaturerule-cpu.yaml

```diff
@@ -0,0 +1,112 @@
+#
```

Comment:
with this Samples, should we remove `examples` ?

Reply:
It is a good question where we should put these 🤔 The `examples/` are really supposed to be examples but these are "fully usable" samples that replicate the built-in labels. Maybe these should be put somewhere under `deployment/` instead, e.g. `deployment/nodefeaturerule/`. Any thoughts?

Reply:
I like `deployment/nodefeaturerule/samples`

Reply:
Let's do that, then. Better I think